test/Microsoft.ML.Benchmarks.Tests/BenchmarksTest.cs

```diff
@@ -41,14 +38,24 @@ public class BenchmarksTest
             Environment.Is64BitProcess; // we don't support 32 bit yet
 #endif
 
+        public static TheoryData<Type> GetBenchmarks()
+        {
+            TheoryData<Type> benchmarks = new TheoryData<Type>();
+            Assembly asm = Assembly.LoadFrom("Microsoft.ML.Benchmarks.dll");
```

Comment:
Can you use `typeof(StochasticDualCoordinateAscentClassifierBench).Assembly` instead?

Reply:
yes sure i will do that but is there any specific reason behind this ?

Reply:
The reason is because the assembly is already loaded, since we have a reference to it. Using a strongly-typed dependency is better because it explicitly shows the dependency, and if someone removed the reference (or renamed the assembly), the code would no longer compile. Also, it is more performant because Assembly.LoadFrom has to interpret the string, read the file, etc. to decide that the assembly is already loaded.